packages/spec-api/src/types.ts

```diff
@@ -48,6 +48,31 @@ export interface MockApi {
   method: HttpMethod;
   uri: string;
   handler: MockRequestHandler;
+  kind: "MockApi";
+}
+
+export interface MockApiDefinition {
+  uri: string;
+  mockMethods: MockMethod[];
+  kind: "MockApiDefinition";
+}
+
+export interface MockMethod {
+  method: HttpMethod;
+  request: ServiceRequest;
+  response: MockResponse;
+  handler?: MockRequestHandler;
+}
+
+export interface ServiceRequest {
+  body?: any;
+  validStatus?: number;
+  params?: {
```

Comment:
are those query params or what kind? can you add a doc on it?

Reply:
Yes. I have added documentation